### PR TITLE
Use Vazir font

### PR DESCRIPTION
**Pull request by pamenary**

### CODE REVIEW

**Comment by pi0** [comment]
Thanks for this PR! License for Vazir seems fine (Public Domain)

Preview: https://deploy-preview-14--vuejs-fa.netlify.com/

We have a problem with letter-spacing:

image

Can you please tweak landing styles?

I think we can also keep this PR for a while gathering community opinions about proper font to choose.

File: themes/vue/source/css/_font.styl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  src: url('../fonts/vazir/Vazir.eot');
+  src: url('../fonts/vazir/Vazir.eot?#iefix') format('embedded-opentype'),
+       url('../fonts/vazir/Vazir.woff2') format('woff2'),
+       url('../fonts/vazir/Vazir.woff') format('woff'),
+       url('../fonts/vazir/Vazir.ttf') format('truetype');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Vazir;
```
Comment (payamnj) [inline themes/vue/source/css/_font.styl:12]: I might be wrong as my CSS knowledge is limited but I think having many different font-face with the same font-family value and just changing the font-weight is not effective and If we want to include the weight we should also change the font-family accordingly. for example `Vazir-Bold`. 
I think we need only the normal font-weight for the font face definition.

**Review by payamnj — COMMENTED** [review]
I like the Vazir font, the letter spacing problem we had with the initial font as well.

File: themes/vue/source/css/_font.styl (hunk 1 above)
Comment (pi0) [reply themes/vue/source/css/_font.styl:12]: It is effective according to font-weight of rendered element:

image

image

Comment (payamnj) [reply themes/vue/source/css/_font.styl:12]: I see, thanks for the explanation.

Comment (pamenary) [reply themes/vue/source/css/_font.styl:12]: Yes, you can do it, but it's not best practice.
it's better different font-face for all font weight

File: themes/vue/source/css/index.styl [hunk 2]
```diff
@@ -62,7 +62,7 @@ body
     margin: 1em 0
     font-size: 1.05em
     font-weight: 600
-    letter-spacing: .1em
+    letter-spacing: 0.04em
```
Comment (pi0) [inline themes/vue/source/css/index.styl]: Why not reset to 0? We still have empty pixels: 

image

**Review by pamenary — COMMENTED** [review]
> Thanks for this PR! License for Vazir seems fine (Public Domain)
> 
> Preview: https://deploy-preview-14--vuejs-fa.netlify.com/
> 
> We have a problem with letter-spacing:
> 
> image
> 
> Can you please tweak landing styles?
> 
> I think we can also keep this PR for a while gathering community opinions about proper font to choose.

Good idea

File: themes/vue/source/css/index.styl (hunk 2 above)
Comment (pamenary) [reply themes/vue/source/css/index.styl]: The page that was rendered to me did not have an empty pixel
But now letter-spacing has been deleted

**Pull request merged**